### PR TITLE
Create documentation for installation and usage

### PR DESCRIPTION
**Pull request by nunocosta-casafari**
# Description

Add readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,89 @@
-# flake8-kw-args
 Flake8 plugin to check explicitly passed arguments.
+
+   * [Getting Started](#getting-started)
```
[the remark this answers is not in the document]
Comment (nunocosta-casafari) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -1,2 +1,89 @@
...
+      * [Installation](#installation)
```
[the remark this answers is not in the document]
Comment (nunocosta-casafari) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -1,2 +1,89 @@
...
+```
+
+Now you can run the tests by with tox.
+```bash
+$ tox
+```
+
+All the configured tests and code quality analysis are run.
+This includes linting and type check.
+
```
[the remark this answers is not in the document]
Comment (nunocosta-casafari) [reply README.md]: done

File: README.md [hunk 4]
```diff
@@ -1,2 +1,89 @@
...
+## Tests
+
+Install it with `pip3`:
+
+```bash
+$ pip3 install tox
+```
+
+Now you can run the tests by with tox.
```
[the remark this answers is not in the document]
Comment (nunocosta-casafari) [reply README.md]: done

File: README.md [hunk 5]
```diff
@@ -1,2 +1,89 @@
...
+```bash
```
[the remark this answers is not in the document]
Comment (nunocosta-casafari) [reply README.md]: done

File: README.md [hunk 6]
```diff
@@ -1,2 +1,86 @@
...
+$ pip3 install tox
+```
+
+Now you can run the tests by with `tox`.
+
+```bash
+$ tox
+```
+
```
Comment (dmytrostriletskyi) [inline README.md]: Remove blank line here.

Comment (nunocosta-casafari) [reply README.md]: done

**Pull request merged**